client/src/api/staff.ts

```diff
@@ -14,6 +15,21 @@ import {
 } from './error.ts';
 
 export namespace Staff {
+    export async function add(uid: User['id'], oid: Office['id']): Promise<boolean> {
+        const res = await fetch(`/api/staff?user=${uid}&office=${oid}`, {
+            method: 'POST',
+            credentials: 'same-origin',
+        });
+        switch (res.status) {
+            case StatusCodes.CREATED: return true;
+            case StatusCodes.NOT_FOUND: return false;
+            case StatusCodes.BAD_REQUEST: throw new InvalidInput;
+            case StatusCodes.UNAUTHORIZED: throw new InvalidSession;
+            case StatusCodes.FORBIDDEN: throw new InsufficientPermissions;
+            default: throw new UnexpectedStatusCode;

```

Comment:
```suggestion
            case StatusCodes.CONFLICT: throw new AlreadyMember;
            default: throw new UnexpectedStatusCode;

```
No `AlreadyMember` error exists though, pero you get the point.

Reply:
Nice catch!